### PR TITLE
Better error message when EC2 instance access is not allowed

### PR DESCRIPTION
**Pull request by arjunrn**
Signed-off-by: Arjun Naik <arjun.rn@gmail.com>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: piu/error_handling.py [hunk 1]
```diff
@@ -47,6 +54,13 @@ def wrapper():
                     file=sys.stderr,
                 )
                 sys.exit(1)
+            elif is_permissions_error(e):
+                print(
+                    "Are you allowed to use SSH access in this account. If this is a Kubernetes account"
```
Comment (aryszka) [inline piu/error_handling.py]: questionmark?

**Comment by aryszka** [comment]
:+1:

File: piu/error_handling.py (hunk 1 above)
Comment (arjunrn) [reply piu/error_handling.py]: Updated

**Comment by arjunrn** [comment]
👍

**Comment by aryszka** [comment]
:+1:

**Pull request merged**